pandas/types/concat.py

```diff
@@ -154,7 +157,7 @@ def is_nonempty(x):
     return np.concatenate(to_concat, axis=axis)
 
 
-def _concat_categorical(to_concat, axis=0):
+def _concat_categorical(to_concat, axis=0, _union_categoricals=False):
```

Comment:
because it has to refer ``union_categoricals`` function defined in ``concat.py``.